.circleci/config.yml

```diff
@@ -0,0 +1,205 @@
+## Required ENV variables in CircleCI account:
+# PACKET_API_TOKEN
+# PACKET_PROJECT_ID
+# PACKET_SSH_PUB_KEY
+
+# Default YAML anchor, re-used in all CircleCI jobs
+_job_defaults: &job_defaults
+  working_directory: ~/ova
+
+# Re-used in jobs to terminate the Packet bare-metal device
+_destroy_device: &destroy_device
+  name: Destroy metal device on failure
+  command: ansible-playbook -v -i inventory destroy.yml
+  working_directory: .circleci/ansible/
+
+
+version: 2
+jobs:
+  # Run Packer Lint checks for OVA json file
+  packer-lint:
+    <<: *job_defaults
+    docker:
+      - image: hashicorp/packer
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: apk add --update make curl
+      - run:
+          name: Packer Lint Check
+          command: make validate
+
+  # Run Inspec Lint checks for test files
+  inspec-lint:
+    <<: *job_defaults
+    docker:
+      - image: chef/inspec:1
+    steps:
+      - checkout
+      - run:
+          name: Inspec Lint Check
+          command: |
+            cd test/integration/
+            for dir in */; do
+              dir=$(basename $dir)
+              if [ -f "${dir}/inspec.yml" ]; then
+                echo -e "\nRunning Inspec lint for \033[1;36m${dir}\033[0m ..."
+                inspec check --diagnose ${dir}
```

Comment:
Cosidering we're moving linting task to Packer, good opportunity to move InSpec linting task into `Makefile` as well.

Reply:
Also it's "Not Important".

I'm OK if we you want to do it later, not a blocker for this PR.

Reply:
I say we do in a future PR.

Reply:
Works for me :+1: